presentation-api/controlling-ua/defaultRequest_success-manual.html

```diff
@@ -1,43 +1,62 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Presentation API, testing to start a new presentation with an default request setup. (success - manual)</title>
+<title>[Optional] Starting a presentation from the browser using a default presentation request. (success - manual)</title>
```

Comment:
Not a big deal, but any particular reason to keep "(success - manual)" in this title? They don't mean much to me at least.